pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -373,6 +373,15 @@ def test_setitem_string_column_numpy_dtype_raising(self):
         expected = DataFrame([[1, 2, 5], [3, 4, 6]], columns=[0, 1, "0 - Name"])
         tm.assert_frame_equal(df, expected)
 
+    def test_setitem_empty_df_duplicate_columns(self):
```

Comment:
do we need tests for Series?

Reply:
Don't think so, indexer has to be a tuple to land there, which is only valid for dataframes I think?

Reply:
its weird, but you _could_ do `series.loc[("foo",)]`

Reply:
Yep, definitely weird :)

But this is already converted before reaching that point.

```
df = Series(index=["a", "b", "b"], dtype="float64")
df.loc[("a", )] = 1
```
This results in ``indexer=0``. Tested the same with a MultiIndex Series, also 0 there